SoftLayer/CLI/registration/contacts.py

```diff
@@ -0,0 +1,40 @@
+"""Get Contact details for a subnet registration."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """Display the RIR contact information for your account.
+
+    """
+    mgr = SoftLayer.NetworkManager(env.client)
+    result = mgr.get_registration_details()
+    table = formatting.Table(['id', 'name', 'email', 'label', 'address', 'phone'])
+    for contact in result:
+        json_result = {'EMAIL_ADDRESS': '', 'INTERNAL_LABEL': ''}
+        json_result['id'] = contact['id']
+        for property_contact in contact['properties']:
+            if property_contact['propertyType']['keyName'] == 'EMAIL_ADDRESS':
+                json_result['EMAIL_ADDRESS'] = property_contact['value']
+                continue
+            elif property_contact['propertyType']['keyName'] == 'INTERNAL_LABEL':
+                json_result['INTERNAL_LABEL'] = property_contact['value']
+                continue
+            json_result[property_contact['propertyType']['keyName']] = (property_contact['value'])
+
+        table.add_row([json_result['id'],
```

Comment:
instead of `json_result['id]` use `json_result.get('id')` for this and all other fields in json_result. Due to the way the registration stuff is designed, I think its certainly possible for  some of these fields to just not exist, which would cause an exception for the user.